lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        defined?(@queue) ? @queue : Disc.default_queue
```

Comment:
Hmm, wouldn't `@queue || Disc.default_queue`suffice? The reason I kept it out of the options hash is to cleanup logic when merging the enqueue options hash with the default disc_options hash.